modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -35,6 +35,20 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172458. User can access Deal Registration Listing through Quick Links"
+	@priority = "4"
+	test CanAccessDealListingViaQuickLinks {
+		task ("When I click on 'MDF' button, on Quick Links section") {
```

Comment:
Hi @juliosilveira77, could you please rename this task to: "When I click on 'Deal Registration' button, on Quick Links section"?